yb-voyager/cmd/exportSchema.go

```diff
@@ -122,3 +133,25 @@ func clearSchemaIsExported(exportDir string) {
 	flagFilePath := filepath.Join(exportDir+"metainfo", "flags", "exportSchemaDone")
 	os.Remove(flagFilePath)
 }
+
+func createMetaInfoJsonFile() {
+	data := MetaInfo{
+		SourceDBType:    source.DBType,
+		SourceDBName:    source.DBName,
+		SourceDBSchema:  source.Schema,
+		SourceDBVersion: source.DB().GetVersion(),
```

Comment:
i think we should consider adding `DBSid` and `TNSAlias` to this metainfo and analyze schema report also, in case of oracle.
cc @rahulb-yb